pipelines/batch/src/main/java/com/google/fhir/analytics/FetchSearchPageFn.java

```diff
@@ -262,6 +273,9 @@ protected void processBundle(Bundle bundle, @Nullable Set<String> resourceTypes)
       numFetchedResources.inc(bundle.getEntry().size());
       if (parquetUtil != null) {
         long startTime = System.currentTimeMillis();
+        // TODO: The right way for writing Parquet records is to cache them and wait
```

Comment:
I guess right, similar problems might also rise while uploading to the sink databases and FHIR servers below, unless they are idempotent.

Reply:
It is a little bit different when writing to a sink DB or FHIR-server if we are careful to just override the same resource. For example, when creating rows from `ViewDefinition` we first delete all rows with the same `id` (which prevents the duplication problem). In any case, I clarified this comment to reflect potential challenges with non-Parquet output as well.